code-samples/serving/gitwebhook-go/Dockerfile

```diff
@@ -32,7 +32,23 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o /go/bin/webh
 
```

Comment:
The GH dependency is probably now hosted here: https://github.com/google/go-github

Reply:
Same fixes applied to _secrets-go_ project, no more build error and the server starts!